### ISSUE TITLE
f-droid ?

### ISSUE THREAD

**Issue by DearFox** [PriyavKaneria/LocalLock#1]
How about publishing your app to f-droid?

**Comment by PriyavKaneria** [comment]
Wasn't aware of it. Looks pretty good, I'll check it out
Is there support for expo builds in fdroid? Looks like I might need to have the native code exported from expo for fdroid to build

**Comment by DearFox** [comment]
> Wasn't aware of it. Looks pretty good, I'll check it out Is there support for expo builds in fdroid? Looks like I might need to have the native code exported from expo for fdroid to build

You must provide f-droid with your code, and they will compile it themselves when updating the program.  This is something like a guarantee that the compiled APK does not contain anything third-party.  But yes, you may need to make changes necessary for f-droid to work.  To be honest, I don’t really know which ones exactly, but I think you can find out the requirements in their documentation.

**Comment by PriyavKaneria** [comment]
Cool I'll try it out

**Comment by PriyavKaneria** [comment]
I went through their docs
https://gitlab.com/fdroid/rfp/-/issues/240

```
tldr;
- author has never been asked for inclusion (at least I can find no matching issue upstream
- comes with a bunch of binaries
- comes with proprietary components (GMS, Facebook, Crashlytics)
- uses untrusted maven repos (fabric)
```

Seems like Expo projects are not supported due to these reasons on F-droid
So sadly I won't be able to deploy it there

**Comment by PriyavKaneria** [comment]
Closing this for now then
